### PR TITLE
Demo with usage of EV

### PR DESCRIPTION
**Pull request by amfrac**
This version needs to use v.1.0.1 of home-energy-managment package which will be released after merging EV simulator into main branch

### CODE REVIEW

File: demo_runner.py [hunk 1]
```diff
@@ -286,6 +319,13 @@ def set_temp_outside(temp: float):
     temp_outside_sensor.set_temp(temp)
 
 
+def set_ev_driving(driving_power: float, time_until_charged_h: Optional[float] = None):
```
Comment (mateuszkobak) [inline demo_runner.py:322]: I am having the impression that there are two different things here and I suggest handling them differently.

1. Setting the driving power of the vehicle. This should be done directly through a method of the EV simulator.
2. Setting *user preferences* describing when the vehicle should be charged and ready to drive. This should be set in the user app. I understand that such user preferences mean nothing to EV (simulator or real-life) as they are the input to the decision-algo and only after executing it we can get the result that is send to the EV. Do I understand it correctly?

**Review by mateuszkobak — CHANGES_REQUESTED** [review]

File: scenario/__pycache__/config.cpython-310.pyc
[the remark this answers is not in the document]
Comment (amfrac) [reply scenario/__pycache__/config.cpython-310.pyc:1]: Sorry, my mistake - I removed it in newest commit

File: scenario/scenario_autumn.py [hunk 2]
```diff
@@ -76,3 +76,11 @@
     (22 * HOUR, [6, 0, 0]),
     (23 * HOUR, [6, 0, 0]),
 ]
+
+EV_POWER_CONFIG = [
```
[the remark this answers is not in the document]
Comment (amfrac) [reply scenario/scenario_autumn.py:80]: Yes, you understand it correctly. Between 22:00 and 0:00 it should see the next non-zero power at 8am, but it didn't work like that - now it's corrected by method get_time_until_departure.

File: demo_runner.py (hunk 1 above)
Comment (amfrac) [reply demo_runner.py]: Okay, I understand what you mean and agree - yes, planned departure time of EV (represented by time_until_charged_h here) has nothing to do with EV simulator so I separated it and changed how it is provided to decision algo.

File: README.md [hunk 3]
```diff
@@ -90,3 +90,8 @@ In *live* mode additionally one can use:
 * `set_consumption(current: float)` – sets auto-consumption (use positive values for consumption)
 
 * `set_temp_outside(temp: float)` – sets temperature outside
+
+* `set_ev_driving(driving_power: float, time_until_charged_h: Optional[float])` - sets EV driving power and optionally
```
Comment (mateuszkobak) [inline README.md:94]: Please update README according to new changes

File: scenario/scenario_autumn.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mateuszkobak) [reply scenario/scenario_autumn.py:80]: How does new changes affect the scenario files?

**Comment by mateuszkobak** [comment]
I left two more comments. Apart from that lgtm :)

File: scenario/scenario_autumn.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mateuszkobak) [reply scenario/scenario_autumn.py:80]: Ok, so maybe you can leave a comment about it there (probably `scenario_autumn` is the one with comments)

File: README.md (hunk 3 above)
Comment (amfrac) [reply README.md]: Done

**Comment by mateuszkobak** [comment]
Please:
- squash commits before you merge
- add new reference to HEM in `requirements.txt`
- test if everything works

**Review by mateuszkobak — APPROVED** [review]
Approved

**Pull request merged**